mantid/wfm/wfm_stitching.md

```diff
@@ -0,0 +1,169 @@
+1. This raw background value is modified in a conservative way by adding to it 5% of the range between the raw background amplitude and the maximum amplitude in the data. This updated background value is represented by the pink dashed line in the right panel.
+1. The leading (left) edge of the signal is then found by iterating through the spectrum, starting from the left hand side and finding the first data point that exceeds the updated background value. This edge is marked by a blue dot in the left panel. The same is repeated for the trailing edge (yellow dot), starting from the right hand side and iterating towards the left.
+1. We then need to find 5 valleys (or inverted peaks) between these two edges. A [peak-finding](https://docs.scipy.org/doc/scipy/reference/generated/scipy.signal.find_peaks.html) algorithm from the Scipy library is called on the inverted data (to find the valleys). The `prominence` parameter is given a value of `0.05 * (signal_maximum - signal_minimum)`. The 5 valleys are marked with red dots in the left panel.
+1. Once the 5 valleys, and hence the 6 frames, are found, we compute the mean amplitude inside each frame. These are represented by the green horizontal lines.
+1. The inter-frame gaps are cut out by defining a frame edge as the first value, starting from the centre of a given valley, that exceeds 30% of the frame average (brown horizontal lines). The frame edges are found by starting from each of the five red dots and navigating through the spectrum towards first the left, to find the previous frame's upper bound, and then the right, to find the next frame's lower bound.
+1. The leading (blue dot) and trailing (yellow dot) edges complete the set of frame boundaries.
+
+![peak-finding](peak_finding.png)
+**Figure 1:** Left: raw data from a diffraction experiment at V20, showing the 6 WFM frames in different colours, as well as various markers that illustrate the steps in the frame finding algorithm (see text). Right: Histogramming of counts in amplitude bins to detect the background as the most common count occurence.
+
+**Links**
+
+- The software that implements Method 1 can be found here: https://git.esss.dk/wedel/wfm_stitching.
+- Contact: Neil Vaytet (ESS/DMSC), Owen Arnold (STFC, Tessella)
+
+### 2.2 Applying the conversion to time-of-flight
+
+Because WFM is essentially making 6 new pulses from a single long pulse, the position of the neutron source (or source chopper in the case of V20) is considered to be half-way between the two WFM choppers. In order to conserve the wavelength of the neutrons, the data needs to be converted  from 'Arrival time at detector' to real 'Time-of-flight'.
+
+Each frame needs to be shifted by a different value, as described in [Woracek et al. (2016)](https://doi.org/10.1016/j.nima.2016.09.034). This article also outlines the method used to compute the different frame shifts, and this will thus not be repeated here. We simply list the numbers by which the frames were shifted:
```

Comment:
The calculation is trivial, but is incorrect in the paper your cite. I think it is worth providing here.

Reply:
Ah, I didn't realise that. Do you have the correct calculation written down somewhere or should i extract it from the code?

Reply:
What is done here is correct https://git.esss.dk/wedel/wfm_stitching/-/blob/calculate_relative_frame_shifts/wfm_stitching/wfm_processing.py#L24-69 that was just a correction of the formula in robins paper

Reply:
So I suggest taking (3) from Robin's paper but correcting. I think it's just the division by 360 that's wrong, should be multiplied.